src/CoreBundle/Exception/SessionAccessFailedException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
```

Comment:
Could you add the default license header?

Reply:
ok